lib/FrontendTool/FrontendTool.cpp

```diff
@@ -504,10 +504,17 @@ createOptRecordFile(StringRef Filename, DiagnosticEngine &DE) {
   return File;
 }
 
+struct PostSILGenInputs {
+  std::unique_ptr<SILModule> TheSILModule;
+  bool astGuaranteedToCorrespondToSIL;
```

Comment:
Please be consistent about initial case across struct fields.

Reply:
Changed to AST...